src/betterproto/streams/__init__.py

```diff
@@ -0,0 +1,82 @@
+from .. import Message, encode_varint, decode_varint
+from io import BufferedReader, BufferedWriter
+from typing import Iterable, Optional
+
+
+class CodedOutputStream:
+    stream: BufferedWriter
+
+    def __init__(self, stream: BufferedWriter) -> None:
+        self.stream = stream
```

Comment:
I think you can just use a dataclass here
```suggestion
@dataclasses.dataclass(frozen=True)
class CodedOutputStream:
    stream: BufferedWriter
```

Reply:
Fixed